### PR TITLE
set encoding for tests on zos

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build spawn.zos=true

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.ws.microprofile.mpjwt.1.1_fat_tck/.classpath.orig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+<<<<<<< HEAD
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.mpjwt.1.1_fat_tck/.classpath.orig]: you don't want this file in here

**Comment by wraschke** [comment]
The com.ibm.ws.microprofile.reactive.streams.operators_fat_tck bucket, introduced under 5338,  is causing https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.workitem.viewWorkItem&id=258957, which has appeared in five builds in just the short amount of time this FAT has been running in master (a day or so). I tried to revert just 5338, but given that this PR adds code on top of 5338's commit, I need to revert this first.

**Pull request merged**